docs/howto/operate/new-cluster/azure-or-google-cloud.md

```diff
@@ -0,0 +1,198 @@
+# Add a new Azure or Google Cloud cluster
```

Comment:
If we imagine the AWS deployment information to be very similar, maybe we should just leave this as `Add a new Kubernetes Cluster`  and not have it name specific clouds. And we can add a `TODO` item to fill out the AWS-specific information.

Reply:
Done across [`7e7fbe0`](https://github.com/2i2c-org/infrastructure/pull/836/commits/7e7fbe051ad619158a6e235d4957faa97ac86d80) and [`26c3b2b`](https://github.com/2i2c-org/infrastructure/pull/836/commits/26c3b2b484ff9afdb3ff1dc4de193c6184361682)